### ISSUE TITLE
Decide how to store talks

### ISSUE THREAD

**Issue by bradgarropy** [remix-austin/new-remixaustin-com#16]
Much like #15, we need to figure out where and how to store talks.

- database
- cms
- local files

**Comment by sethdavis512** [comment]
What information are we wanting to store with the talks? Title, video url and small blurb? If that's the case maybe just a JSON file for now?

**Comment by bradgarropy** [comment]
I'd be fine with a JSON implementation. Do we have data for past talks? Is there any way we could just source it directly from meetup.com?

**Comment by sethdavis512** [comment]
I don't think we have it but I think creating 3-5 previous events within a JSON file should suffice. We can have a little banner or link pointing anyone who's interested in events further back to https://www.meetup.com/remix-austin/events/?type=past
